domain-models-runtime/src/main/java/org/folio/rest/persist/cql/CQLWrapper.java

```diff
@@ -150,25 +150,56 @@ private void spaceAppend(StringBuilder sb, String text) {
     sb.append(text);
   }
 
-  /**
-   * @return a space followed by the SQL clauses of WHERE, OFFSET and LIMIT
-   * @throws CQLQueryValidationException when the underlying CQL2PgJSON throws a QueryValidationException
-   */
-  @Override
-  public String toString() {
+  public String getWhere() {
     StringBuilder sb = new StringBuilder();
-    append(sb, query, field);
+    appendWhere(sb, query, field);
     for (WrapTheWrapper wrap : addedWrappers) {
       if (sb.length() > 0) {
         // (q1) operator q2 .. left-associative
         sb.insert(0, '(');
         sb.append(") ").append(wrap.operator).append(' ');
       }
-      append(sb, wrap.wrapper.getQuery(), wrap.wrapper.getField());
+      appendWhere(sb, wrap.wrapper.getQuery(), wrap.wrapper.getField());
     }
-    if (sb.length() > 0) {
-      sb.insert(0, " WHERE ");
+    return sb.toString();
+  }
+
+  public String getWhereFull() {
+    String s = getWhere();
+    if (!s.isEmpty()) {
+      return " WHERE " + s;
+    }
+    return "";
+  }
+
+  public String getOrderBy() {
+    if (query == null || field == null) {
+      return "";
+    }
+    try {
+      return field.toSql(query).getOrderBy();
+    } catch (QueryValidationException e) {
+      throw new CQLQueryValidationException(e);
     }
+  }
+
+  public String getOrderByFull() {
+    String s = getOrderBy();
+    if (!s.isEmpty()) {
+      return "ORDER BY " + s;
+    }
+    return "";
```

Comment:
More easy to read and understand and avoids the negation:

    if (s.isEmpty()) {
      return "";
    }
    return "ORDER BY " + s;

Reply:
ok